cg/store/models.py

```diff
@@ -436,6 +436,14 @@ def latest_ticket(self) -> str | None:
         """Returns the last ticket the family was ordered in"""
         return self.tickets.split(sep=",")[-1] if self.tickets else None
 
+    @property
+    def latest_order(self) -> "Order":
+        """Returns the latest order this case was included in."""
+        sorted_orders: list[Order] = sorted(
+            self.orders, key=lambda order: order.order_date, reverse=True
+        )
+        return sorted_orders[0]
```

Comment:
Aren't new cases created with each order? So a case would always have the "latest_order" associated with it?

Reply:
You can include old cases in new orders, which would mean that we would have a list of orders associated with the case. And we can not be sure that we start out with an order that can be passed along alongside the case, since we can run a `cg workflow <pipe_line> start <case_id>` command, upon which we need to choose which order it should be added to.